applications/portal/frontend/src/components/sidebar/PopulationsAccordion.jsx

```diff
@@ -128,6 +129,7 @@ const PopulationsAccordion = ({
                                 <Switch />
                             }
                             onChange={handleShowAllPopulations}
+                            disabled={areAllPopulationsWithChildrenCells(populations)}
```

Comment:
Why can't we just check if populations is empty?

Reply:
![image](https://github.com/MetaCell/salk-interactive-atlas/assets/19196034/8223aea3-76e0-42b6-a6ae-79fcde04bdc4)

Reply:
@afonsobspinto what about if the case when all the children switches are disabled?